content/python/working_with_user_accounts.md

```diff
@@ -0,0 +1,406 @@
+---
+title: "Working with User Accounts"
+description: "A few examples on interacting with User Accounts"
+date: "2022-01-03"
+classes: 
+    - "SoftLayer_Account"
+tags:
+    - "User"
+    - "Account"
+    - "User_Account"
+---
+
+### Setup
+All the functions defined in this article will be part of this `UserAccount` class. Which only sets up the SoftLayer Client, and configures the debugger, which allows you to see the exact API calls being made.
+
+```python
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from pprint import pprint
+class UserAccount:
```

Comment:
I think there is a missing blank line at the beginning of the class and between each function

Reply:
The blank lines were added at the beginning of the class and between each function